examples/sample_config.yml

```diff
@@ -1,3 +1,4 @@
+# editorconfig-checker-disable-file
```

Comment:
I think all our yml should follow the same editorconfig

Reply:
We would need to define a new configuration format to support. See the error below that occurs when trying to match the editorconfig formatting in our unit tests.

```bash
---- config::raw::test::full_deserialize stdout ----
thread 'config::raw::test::full_deserialize' panicked at src/config/raw.rs:492:63:
called `Result::unwrap()` on an `Err` value: Error { kind: SCANNER, problem: "mapping values are not allowed in this context", problem_mark: Mark { line: 9, column: 18 } }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I'm fine with updating the configuration format such that we can meet the formatter, but that would need to be a v2 release.

Reply:
I think this just means we need to get our yaml setting sorted in our .editorconfig right?

Reply:
I can try and dig into editorconfig some more, this is my first experience with it. But I didn’t see controls like that.